Profiler/install.cmd

```diff
@@ -0,0 +1,79 @@
+@echo off
+setlocal EnableDelayedExpansion
+
+rem check for admin rights
+net session >nul 2>&1
+if %errorLevel% NEQ 0 (
+	echo Failure: Execute script as Administrator.
+	pause
+	exit /B 1
+)
+
+
+set InstallSource=%CD%
+for /f "delims=" %%x in (%InstallSource%\install.cfg) do (
+	set line=%%x
+	if "!line:~0,1!" neq "#" (
+		set !line!
+	)
+)
+
+if "%InstallDir%"=="" (
+	echo Failure: InstallDir not set.
+	pause
+	exit /B 1
+)
+
+echo Installing Profiler to: "%InstallDir%"
+
+if exist "%InstallDir%" (
+	if "%InstallUploadService%"=="true" (
```

Comment:
do we really need this? what if the user previously installed in a different directory? then this will do nothing and installing the service will still fail later on. I'd rather just not support the re-install case and instead add an uninstall.ps1 file that removes the daemon and prompts the user to manually delete the folder.